evm/src/main/scala/pravda/evm/translate/Translator.scala

```diff
@@ -0,0 +1,28 @@
+package pravda.evm.translate
+
+import pravda.evm.EVM
+import pravda.evm.translate.opcode.SimpleTranslation
+import pravda.vm.asm
+
+object Translator {
+
+
+  private def listToEither[L,R](eithers: List[Either[L,List[R]]]): Either[List[L],List[R]] = {
```

Comment:
this is `.sequence` method from cats, `cats.core` is our global dependecy

Reply:
Here I need to aggregate all error messages from either. As far as I know, `.sequence` from cats doesn't provide that.